gneiss/_summary.py

```diff
@@ -57,3 +58,72 @@ def __init__(self, stat_results,
         # calculate the overall coefficient of determination (i.e. R2)
         sst = sse + ssr
         self.r2 = 1 - sse / sst
+
+    def _check_projection(self, project):
```

Comment:
OK, thanks. Just out of curiosity, will it make sense to make project a member of the object? What I'm asking is if it will make sense to do somewhere in the code: self.project=True/False, and then just call self.coefficients() & self._check_projection(). This could simplify the code but it will mean that if you are doing self.residuals, self.predict, etc this parameter can't change.